spec/presenters/hyku/manifest_enabled_work_show_presenter_spec.rb

```diff
@@ -25,4 +25,25 @@
       expect(subject).to be_kind_of Hyku::FileSetPresenter
     end
   end
+
+  describe "manifest_extras" do
+    subject do
+      presenter.manifest_extras
+    end
+
+    let(:work) { FactoryGirl.create(:work_with_one_file) }
+    let(:document) { work.to_solr }
```

Comment:
Ideally, we'd do this once ahead of time and avoid having to persist this work just so we can generate a solr document.  This helps keep the test suite fast.